uint32 height
uint32 width
float32 resolution
float64 origin_x
float64 origin_y
float32 explored_ratio
uint16[] mask